modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js

```diff
@@ -228,4 +228,23 @@ describe('DatePicker', () => {
 			).toBeTruthy()
 		);
 	});
+
+	it('fills the input completely when last item of a date mask is a symbol Ex: (YYYY.MM.DD.)', () => {
+		const handleFieldEdited = jest.fn();
+
+		const {container} = render(
+			<DatePickerWithProvider
```

Comment:
I think you can use `<DatePicker />` directly:
```suggestion
			<DatePicker
```